# Object coordinates message
float64[] coordinates
float64[] average